_posts/2021-11-18-release-notes.md

```diff
@@ -182,7 +182,7 @@ USN-5133-1 USN-5133-1: ICU vulnerability:
 
 ### Diego Release 2.53.1 up from 2.53.0
 Changes
-* Bump to Go MySQL Driver v1.6.0 @mariash (#595)
+* Bump Go driver for MySQL Driver to v1.6.0 @mariash (#595)
```

Comment:
Is the issue that it was "Go MySQL Driver" or just because it was "MySQL Driver"?

If it was just the phrase "MySQL Driver", should this be changed to the following?

```suggestion
* Bump Go driver for MySQL to v1.6.0 @mariash (#595)
```